vue.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    baseUrl: './'
```

Comment:
How does `gh-pages` know which gh-pages account to push the changes to? In my other project where I used it, there was an option somewhere to set the actual github.io URL to use.

Reply:
gh-pages pushes to whatever account the repo has as a remote. Is that facility to set the URL important to you or can we leave it as the current account?

The specific change brought in by the vue.config.js here is to have the resources in the index.html document be linked by relative paths when built into dist, otherwise they weren't being found as a part of github.io pages. FYI only.

Reply:
I guess the thing now is for me to switch to this branch and try publishing right? I will go ahead and do that, and then I'll be ready to merge this.